src/Sylius/Bundle/AdminBundle/Resources/views/Product/_info.html.twig

```diff
@@ -1,11 +1,9 @@
-{% set product = variant.product %}
-
 <div class="ui header">
     {% include '@SyliusAdmin/Product/_mainImage.html.twig' with {'product': product, 'filter': 'sylius_admin_product_tiny_thumbnail'} %}
     <div class="content">
-        <a href="{{ path('sylius_admin_product_update', {'id': product.id}) }}" class="sylius-product-name" title="{{ product.name }}">{{ product.name }}</a>
-        <span class="sub header sylius-product-variant-code" title="{{ variant.code }}">
-            {{ variant.code }}
+        <div class="sylius-product-name" title="{{ item.immutableProductName }}">{{ item.immutableProductName }}</div>
```

Comment:
Do we need to add `immutable` prefix? It is set to order item, so it shouldn't change anyway. 

Just thinking